### ISSUE TITLE
[Accessibility] Narrator incorrect announcement issues in DGV comboBox cell

### ISSUE THREAD

**Issue by John-Qiao** [dotnet/winforms#4504]
* .NET Core Version: 
.NET SDK 6.0.100-preview.1.21077.9

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**

1. After step 4, sometimes Narrator read the changed value twice.
![Issue1](https://user-images.githubusercontent.com/45864985/106101816-68ac6e00-6179-11eb-9542-c8695552fd3c.png)
![Issue1-ReproSteps](https://user-images.githubusercontent.com/45864985/106101822-6ba75e80-6179-11eb-8307-416705d8d914.gif)

2. After step 6, Narrator read four lines messages after selected value using mouse.
![Issue2](https://user-images.githubusercontent.com/45864985/106101830-6f3ae580-6179-11eb-9af8-ecb8c8b863e8.png)
![Issue2-ReproSteps](https://user-images.githubusercontent.com/45864985/106101837-719d3f80-6179-11eb-9afe-7983e58baccc.gif)

**Expected behavior:**

1. After step 4, Narrator should read the changed value once.
2. After step 6, Narrator should read two lines message after selected value using mouse, like it works with .NET 4.7.2.
![Issue2-Expected](https://user-images.githubusercontent.com/45864985/106101700-369b0c00-6179-11eb-82e4-1b4c9eaac468.png)

**Minimal repro:**

1. Open the attached [TestApp.zip](https://github.com/dotnet/winforms/files/5885343/TestApp.zip) project in VS, run it and launch the Narrator.
2. Click Tab key to focus on one DGV comboBox cell.
3. Under edit mode, use arrow key to change value in it.
4. Repeat step 2 more times.
5. Click Tab key to focus on one DGV comboBox cell.
6. Expand comboBox cell and select one value using mouse.

**Comment by SergeySmirnov-Akvelon** [comment]
During investigation, I found that messages about the selected item are generated by this [line](https://github.com/dotnet/winforms/blob/main/src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.ComboBoxAccessibleObject.cs#L242). When commenting it, the Narrator does not announce information that the selected item has changed. Adding logs and listening for events using `AccEvent` tool showed that the event was generated only once. Despite this, the Narrator periodically generates 2 messages about the selected item. 

I have sent feedback on my research results to Narrator's team. Created a [bug](https://microsoft.visualstudio.com/OS/_workitems/edit/34751381) to investigate this feedback by Narrator's team

**Comment by SergeySmirnov-Akvelon** [comment]
Since this bug is related to the Narrator's behavior, we close it as "Won't fix"
